### PR TITLE
Update design of the save bar to match new mockup

### PR DESCRIPTION
**Pull request by gaetanmaisse**
**Issue**

NA

**Description**

Update design of the save bar to match new mockup available on [Figma](https://www.figma.com/file/XvLO9G5fPRIrfyhLjPg6a2/Gravitee_Lib_elemZ?node-id=1221%3A2315).

image

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://6183b02d73381a003a3be1a6-klgnhltchv.chromatic.com)

### CODE REVIEW

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@2.6.2-update-save-bar-6cdfb48
# or
yarn add @gravitee/ui-particles-angular@2.6.2-update-save-bar-6cdfb48
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-policy-studio-angular@2.6.2-update-save-bar-6cdfb48
# or
yarn add @gravitee/ui-policy-studio-angular@2.6.2-update-save-bar-6cdfb48
```

File: ui-particles-angular/projects/ui-particles-angular/src/lib/gio-save-bar/gio-save-bar.component.scss [hunk 1]
```diff
@@ -22,7 +22,7 @@ $typography: map.get(gio.$mat-theme, typography);
 
 :host {
   display: block;
-  height: 50px;
+  height: 68px;
```
Comment (gcusnieux) [inline ui-particles-angular/projects/ui-particles-angular/src/lib/gio-save-bar/gio-save-bar.component.scss:25]: Be careful, with this and integration in console api design

**Review by gcusnieux — APPROVED** [review]

File: ui-particles-angular/projects/ui-particles-angular/src/lib/gio-save-bar/gio-save-bar.component.scss (hunk 1 above)
Comment (gaetanmaisse) [reply ui-particles-angular/projects/ui-particles-angular/src/lib/gio-save-bar/gio-save-bar.component.scss:25]: 👌🏻  Yes, and we need to check with cockpit and @audrey-tlm first

**Review by ThibaudAV — APPROVED** [review]

File: ui-particles-angular/projects/ui-particles-angular/src/lib/gio-save-bar/gio-save-bar.component.html [hunk 2]
```diff
@@ -18,9 +18,9 @@
 
 <div *ngIf="isOpen()" [@slideUpDown] [@.disabled]="creationMode">
   <mat-card class="save-bar__content" [class.mat-elevation-z3]="creationMode" [class.mat-elevation-z6]="!creationMode">
```
Comment (audrey-tlm) [inline ui-particles-angular/projects/ui-particles-angular/src/lib/gio-save-bar/gio-save-bar.component.html:20]: Can we have the elevation level "mat-elevation-z5" in both cases?

```suggestion
  <mat-card class="save-bar__content mat-elevation-z5">
```

File: ui-particles-angular/projects/ui-particles-angular/src/lib/gio-save-bar/gio-save-bar.component.html [hunk 3]
```diff
@@ -18,9 +18,9 @@
...
-    <div class="save-bar__content__label">{{ creationMode ? '' : 'Please note you have unsaved modifications.' }}</div>
+    <div class="save-bar__content__label">{{ creationMode ? '' : 'You have unsaved changes.' }}</div>
     <div>
-      <button *ngIf="!creationMode" class="save-bar__content__reset-button" mat-stroked-button type="button" (click)="onResetClicked()">
+      <button *ngIf="!creationMode" class="save-bar__content__reset-button" mat-button type="button" (click)="onResetClicked()">
         Reset
```
Comment (audrey-tlm) [inline ui-particles-angular/projects/ui-particles-angular/src/lib/gio-save-bar/gio-save-bar.component.html:24]: Button text : "Discard"

File: ui-particles-angular/projects/ui-particles-angular/src/lib/gio-save-bar/gio-save-bar.component.html [hunk 4]
```diff
@@ -18,9 +18,9 @@
 
 <div *ngIf="isOpen()" [@slideUpDown] [@.disabled]="creationMode">
   <mat-card class="save-bar__content" [class.mat-elevation-z3]="creationMode" [class.mat-elevation-z6]="!creationMode">
-    <div class="save-bar__content__label">{{ creationMode ? '' : 'Please note you have unsaved modifications.' }}</div>
+    <div class="save-bar__content__label">{{ creationMode ? '' : 'You have unsaved changes.' }}</div>
```
Comment (audrey-tlm) [inline ui-particles-angular/projects/ui-particles-angular/src/lib/gio-save-bar/gio-save-bar.component.html:21]: Label text: You have changes that are not saved (without the ".")

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-policy-studio-angular@2.6.2-update-save-bar-7c54710
# or
yarn add @gravitee/ui-policy-studio-angular@2.6.2-update-save-bar-7c54710
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@2.6.2-update-save-bar-7c54710
# or
yarn add @gravitee/ui-particles-angular@2.6.2-update-save-bar-7c54710
```

File: ui-particles-angular/projects/ui-particles-angular/src/lib/gio-save-bar/gio-save-bar.component.html (hunk 4 above)
Comment (gaetanmaisse) [reply ui-particles-angular/projects/ui-particles-angular/src/lib/gio-save-bar/gio-save-bar.component.html]: Seen with Lju we will go with `You have unsaved changes`

File: ui-particles-angular/projects/ui-particles-angular/src/lib/gio-save-bar/gio-save-bar.component.html (hunk 2 above)
Comment (gaetanmaisse) [reply ui-particles-angular/projects/ui-particles-angular/src/lib/gio-save-bar/gio-save-bar.component.html:20]: As discussed we have 2 different elevations to match one of the cards when we are in a creation form. Before updating this we need to ensure every cases are well handled. Same thing if we want the default elevation to be 5, it needs to be aligned with the cards.

**Pull request merged**